spec/grub_cfg_spec.rb

```diff
@@ -25,4 +25,29 @@
       )
     end
   end
+
+  describe "#boot_entries" do
+    it "gets boot entries list with title: and path: elements" do
+      expect(config.boot_entries).to eq(
+        [
+          {
+            title: "openSUSE Leap 42.1",
+            path: "openSUSE Leap 42.1"
+          },
+          {
+            title: "openSUSE Leap 42.1, with Linux 4.1.12-1-default",
+            path: "Advanced options for openSUSE Leap 42.1>openSUSE Leap 42.1, with Linux 4.1.12-1-default"
+          },
+          {
+            title: "openSUSE Leap 42.1, with Linux 4.1.12-1-default (recovery mode)",
+            path: "Advanced options for openSUSE Leap 42.1>openSUSE Leap 42.1, with Linux 4.1.12-1-default (recovery mode)"
+          },
+          {
+            title: "halt",
+            path: "halt"
+          }
+        ]
+      )
+    end
+  end
```

Comment:
in fact it is different. Parser is interface, without base class. as interface can be used anything that fits expectation, which is three methods as documented at https://github.com/config-files-api/config_files_api/blob/master/lib/cfa/base_model.rb#L8

I found that also your test is not correct, so I modify it.